pkg/minikube/tunnel/kic/ssh_tunnel.go

```diff
@@ -118,7 +118,7 @@ func (t *SSHTunnel) stopMarkedConnections() {
 
 // sshConnName creates a uniq name for the tunnel, using its name/clusterIP/ports.
 // This allows a new process to be created if an existing service was changed,
-// the new process will support the IP/Ports change ocurred.
+// the new process will support the IP/Ports change occurred.
```

Comment:
could u also fix the other lint error on master?
"
consider running: /home/runner/work/minikube/minikube/hack/boilerplate/fix.sh"
Feel free to make a new PR both lints I will merge quickly.